README.md

```diff
@@ -1,23 +1,23 @@
-![Salesforce Higher Education Data Architecture](https://cloud.githubusercontent.com/assets/1894238/13441500/13d366ec-dfbd-11e5-9df6-d9dee50ce87d.jpg "Salesforce Higher Education Data Architecture")
```

Comment:
It might be a good idea for @aligeez or @juliesnow  to review some of the changes I made in this file ("higher-ed-specific" to "education-specific", etc)

Reply:
There's an open discussion among Julie & Liane in Quip on whether or not we should remove the SRS link.